apix-interface/src/main/java/eu/xenit/apix/comments/Conversation.java

```diff
@@ -0,0 +1,64 @@
+package eu.xenit.apix.comments;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Conversation {
+
+    private List<Comment> comments;
+    private boolean hasMore;
+    private boolean creatable;
+
+    public Conversation() {
+    }
+
+    public Conversation(List<Comment> comments, boolean hasMore, boolean creatable) {
+        this.comments = comments;
+        this.hasMore = hasMore;
+        this.creatable = creatable;
+    }
+
+    public List<Comment> getComments() {
+        return comments;
+    }
+
+    public void setComments(List<Comment> comments) {
+        this.comments = comments;
+    }
+
+    public boolean isHasMore() {
```

Comment:
isHasMore -> hasMore

Reply:
Lars made a note earlier that changing the format might mess with serialization/deserialization? Still requires change, or keep as is?

Reply:
No, you can leave it then if has impact.